app/models/qing_group.rb

```diff
@@ -55,6 +55,9 @@ class QingGroup < FormItem
 
   delegate :standard_copy?, :published?, to: :form
 
+  # Don't require a name for root groups.
+  validates :group_name, presence: true, if: -> { parent.present? }
```

Comment:
`root_group` was causing the double validation. It had an `id` but not a `name`. Thanks for pointing in the right direction when pairing today.